example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml

```diff
@@ -1698,6 +1698,126 @@ spec:
                 under. This is necessary to generate correct URLs. This is necessary
                 if Thanos Ruler is not served from root of a DNS name.
               type: string
+            grpcServerTlsConfig:
+              description: 'GRPCServerTLSConfig configures the GRPC server from which
+                Thanos Querier reads recorded rule data. Note: currently only the
+                CAFile, CertFile, and KeyFile fields are supported. Maps to the ''--grpc-server-tls-*''
+                CLI args.'
+              properties:
+                ca:
+                  description: Stuct containing the CA cert to use for the targets.
+                  properties:
+                    configMap:
+                      description: ConfigMap containing data to use for the targets.
+                      properties:
+                        key:
+                          description: The key to select.
+                          type: string
+                        name:
+                          description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                            TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
TODO? 🤔

Reply:
This is coming from the Kube API, I haven't looked into this too deeply, but will likely go away in the next update of the crd generation tools.